extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -245,7 +245,7 @@ Each feature ID definition may include only a single source, so the following ar
 - `featureId.offset` and `featureId.repeat`
 - `featureId.index`
 
-The `featureIds` and `propertyTables` arrays must have the same length, with the feature ID definition at index `i` corresponding to the property table at the same index. Each `featureIds:propertyTable` pair must be unique, but individual feature IDs and property tables may be repeated within a primitive or node.
+Every `propertyTables` pointer must have an associated `featureIds` definition, but feature IDs may be defined without a property table. The `featureIds` entry at index *i* corresponds to the `propertyTables` entry at the same index, if any. As a result, the length of the `featureIds` array must be greater than or equal to the length of the `propertyTables` array. Each `featureIds:propertyTable` pair must be unique, but individual feature IDs and property tables may be repeated within a primitive or node.
```

Comment:
I would say "index" instead of "pointer" since the former seems more common.

Reply:
again, use code blocks for "index `i`"

Reply:
Hm just noticed, should we use `(featureIds, propertyTable)` instead of the colon? this seems more typical if I'm describing a pair (except perhaps in the context of a dictionary which this is not)

Reply:
All sounds good. Agreed the (a,b) syntax is probably clearer. 👍